packages/desktop/src/renderer/styles/styles.scss

```diff
@@ -122,11 +122,19 @@ input,
 $http-methods: ('get', lighten($blue, 5%)), ('put', lighten($orange, 5%)),
   ('post', lighten($green, 5%)), ('delete', lighten($red, 8%)),
   ('head', lighten($purple, 12%)), ('patch', $teal),
-  ('options', lighten($dark-blue, 15%));
+  ('options', lighten($dark-blue, 15%)),
+  ('propfind', lighten($custom-gray, 43%)), ('proppatch', lighten($custom-gray, 43%)),
+  ('copy', lighten($custom-gray, 43%)), ('move', lighten($custom-gray, 43%)),
+  ('lock', lighten($custom-gray, 43%)), ('unlock', lighten($custom-gray, 43%)),
+  ('mkcol', lighten($custom-gray, 43%));
```

Comment:
@255kb I didn't found any usages of the variable `$http-methods` only for `$http-methods-light`.
But to be sure if I had overlooked a usage I've added the decent darker colors here (and also checked the contrast).

Reply:
It's not used anymore/yet. I usually remove unused things, but I was lazy this time I guess. 
Thank you for updating it.

Reply:
Ok, then I'll remove it together with the linting.